src/test/java/seedu/address/logic/parser/EditPersonCommandParserTest.java

```diff
@@ -90,12 +90,6 @@ public void parse_invalidValue_failure() {
         // is tested at {@code parse_invalidValueFollowedByValidValue_success()}
         assertParseFailure(parser, "1" + PHONE_DESC_BOB + INVALID_PHONE_DESC, Phone.MESSAGE_CONSTRAINTS);
 
-        // while parsing {@code PREFIX_TAG} alone will reset the tags of the {@code Person} being edited,
-        // parsing it together with a valid tag results in error
-        assertParseFailure(parser, "1" + TAG_DESC_FRIEND + TAG_DESC_HUSBAND + TAG_EMPTY, Tag.MESSAGE_CONSTRAINTS);
-        assertParseFailure(parser, "1" + TAG_DESC_FRIEND + TAG_EMPTY + TAG_DESC_HUSBAND, Tag.MESSAGE_CONSTRAINTS);
-        assertParseFailure(parser, "1" + TAG_EMPTY + TAG_DESC_FRIEND + TAG_DESC_HUSBAND, Tag.MESSAGE_CONSTRAINTS);
-
```

Comment:
Why this test case need to remove?

Reply:
The original implementation rejects the whole command if it is t/abc t/
Where abc is an arbitrary tag name

With the easing of command syntaxes, doing t/abc t/ would just ignore the t/ behind and accept the command (user would only have the "abc" tag)

Since its not a parse failure, the test cases don't match the new behaviour so they are removed entirely